tests/consensus_tests/auth_tests/test_dashboard_whitelist_bypass.py

```
@@ -0,0 +1,92 @@
+    finally:
+        kill_all_processes()
+
+
+def test_dashboard_prefix_is_whitelisted(dashboard_cluster):
+    """Precondition: the dashboard itself loads without an API key.
+
+    This guards the regression test below: if the /dashboard prefix were not
+    whitelisted here, the bypass test could not distinguish a fixed build from a
+    vulnerable one.
+    """
+    response = requests.get(f"{REST_URI}/dashboard")
```

Ruff (0.15.14)

[error] Probable use of `requests` call without timeout

(S113)